src/Parser/SimpleArrayParser.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace MetaHydrator\Parser;
+
+
+class SimpleArrayParser extends AbstractParser
+{
+    /**
+     * @param $rawValue
+     * @return mixed|null
+     * @throws \MetaHydrator\Exception\ParsingException
+     */
+    public function parse($rawValue)
+    {
+        if ($rawValue === null || $rawValue === '') {
```

Comment:
```suggestion
        if ($rawValue === null) {
```

There is no need to check against empty string, this should be invalid (i.e it should throw an exception)